scripts/portal.json

```diff
@@ -161,29 +161,151 @@
+                    },
+                    {
+                        "name":"plot",
+                        "type":"int"
+                    },
+                    {
+                        "name":"stack",
+                        "type":"int"
+                    },
+                    {
+                        "name":"species",
+                        "type":"char"
+                    },
+                    {
+                        "name":"abundance",
+                        "type":"int"
+                    }
+                ]
+            },
+            "url": "https://github.com/weecology/PortalData/blob/master/Ants/Portal_ant_bait.csv"
+        },
+        {
+            "dialect": {
+                "delimiter":",",
+                "missingValues":[
+                    "",
+                    null
+                ]
+            },
+            "name": "portal_ant_colony",
+            "schema": {
+                "fields":[
+                    {
+                        "name":"day",
+                        "type":"int"
+                    },
+                    {
+                        "name":"month",
+                        "type":"int"
+                    },
+                    {
+                        "name":"year",
+                        "type":"int"
+                    },
+                    {
+                        "name":"plot",
+                        "type":"int"
+                    },
+                    {
+                        "name":"stack",
+                        "type":"int"
+                    },
+                    {
+                        "name":"species",
+                        "type":"char"
+                    },
+                    {
+                        "name":"colonies",
+                        "type":"int"
+                    },
+                    {
+                        "name":"openings",
+                        "type":"int"
+                    },
+                    {
+                        "name":"flag",
+                        "type":"int"
+                    }
+                ]
+            },
+            "url": "https://raw.githubusercontent.com/weecology/PortalData/master/Ants/Portal_ant_colony.csv"
+        },
+        {
+            "dialect": {},
+            "name": "plots_ant_dataflags",
+            "schema": {},
+            "url": "https://github.com/weecology/PortalData/blob/master/Ants/Portal_ant_dataflags.csv"
+        },
+        {
+            "dialect": {
+                "delimiter":",",
+                "missingValues":[
+                    "",
+                    null
+                ]
+            },
+            "name": "plots_ant_species",
+            "schema": {},
+            "url": "https://github.com/weecology/PortalData/blob/master/Ants/Portal_ant_species.csv"
         }
     ],
     "retriever": "True",
     "retriever_minimum_version": "2.0.dev",
     "scan_lines": "40000",
```

Comment:
Should i remove value or update it?

I was looking for `"scan_lines"` in documentation. There is not any mention of it. Do we need it to include in documentation.